packages/react-devtools-shell/webpack.config.js

```diff
@@ -111,9 +110,29 @@ if (TARGET === 'local') {
   config.devServer = {
     hot: true,
     port: 8080,
-    clientLogLevel: 'warning',
-    publicPath: '/dist/',
-    stats: 'errors-only',
+    static: {
+      directory: resolve(__dirname, 'dist'),
+    },
+    client: {
+      overlay: {
+        errors: true,
+        warnings: false,
```

Comment:
@ScriptedAlchemy Why warnings were disabled?

Reply:
I think he's just copying over the behavior from my branch, but this is a good point. We should probably enable `warnings` too.